src/app/shared/components/template/components/youtube/youtube.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from "@angular/core";
+import { TemplateBaseComponent } from "../base";
+import { DomSanitizer, SafeResourceUrl } from "@angular/platform-browser";
+import {
+  getBooleanParamFromTemplateRow,
+  getStringParamFromTemplateRow,
+} from "src/app/shared/utils";
+import { TemplateTranslateService } from "../../services/template-translate.service";
+
+interface IYoutubeParams {
+  /** TEMPLATE PARAMETER: video_id */
+  videoId: string;
```

Comment:
nit(blocking): I don't think we need to have both `value` and `parameter_list` used to define the video id, I'd say better to just keep as `value`